R/utils_gcp.R

```diff
@@ -0,0 +1,105 @@
+# Semi-automated tests of Google Cloud Storage integration live in tests/gcp/. # nolint
+# These tests should not be fully automated because they
+# automatically create S3 buckets and upload data,
+# which could put an unexpected and unfair burden on
+# external contributors from the open source community.
+# nocov start
+gcp_gcs_exists <- function(key,
+                           bucket = gcp_gcs_bucket(),
+                           version = NULL) {
+  tryCatch(
+    gcp_gcs_head_true(
+      key = key,
+      bucket = bucket,
+      version = version
+    ),
+    error = function(condition) {
+      FALSE
+    }
+  )
+}
+
+# to keep lines shorter
+gcp_gcs_bucket <- function() {
+  googleCloudStorageR::gcs_get_global_bucket()
+}
+
+gcp_gcs_head <- function(
+  key,
+  bucket = gcp_gcs_bucket(),
+  version = NULL
+) {
+  suppressMessages(
+    googleCloudStorageR::gcs_get_object(
+      key,
+      bucket = bucket,
+      meta = TRUE,
+      generation = version
+    )
+  )
+
```

Comment:
Would you remove blank lines inside each function body? I feel these functions are concise enough to not need them.